src/services/NavigationService.ts

```diff
@@ -1,10 +1,18 @@
-import { NavigationContainer } from '@react-navigation/native';
-import React from 'react';
+import { NavigationContainerRef, NavigationAction, NavigationState } from '@react-navigation/native';
+import { createRef } from 'react';
 
-export const navigationRef: any = React.createRef<typeof NavigationContainer>();
+export const navigationRef = createRef<NavigationContainerRef>();
 
 export default class NavigationService {
-  navigate = (name: string, params: unknown) => {
-    navigationRef.current?.navigate(name, params);
-  };
+  goBack() {
+    navigationRef.current?.goBack();
+  }
+  navigate(routeName: string, params?: Record<string, any>) {
```

Comment:
![Zrzut ekranu 2020-11-20 o 11 16 31](https://user-images.githubusercontent.com/67273035/99788797-efadec00-2b21-11eb-92bd-301425c6588f.png)
with this we will have routeName and params typescript validation.

Remove: Unlock, DeleteWallet, ReleaseNotes from Routes enum. They are not unused, someone had to forgot to delete them when deleting screen